sources/packages/web/src/services/ApplicationOfferingChangeRequestService.ts

```diff
@@ -190,7 +191,9 @@ export class ApplicationOfferingChangeRequestService {
    */
   async updateApplicationOfferingChangeRequestStatus(
     applicationOfferingChangeRequestId: number,
-    payload: StudentApplicationOfferingChangeRequestAPIInDTO,
+    payload:
```

Comment:
I would discourage using the same service method here. Both ministry and student operations have different DTO and there is no common component using this method.